### PR TITLE
Added support for Tuya's _TZE200_abatw3kj named TS0601_RTX_DIN

### PR DESCRIPTION
**Pull request by vitalybibikov**

### CODE REVIEW

**Comment by vitalybibikov** [comment]
Image added in https://github.com/Koenkk/zigbee2mqtt.io/pull/3009

**Comment by vitalybibikov** [comment]
npm run pretty:write        

> zigbee-herdsman-converters@20.12.1 pretty:write
> prettier --write .

.devcontainer/devcontainer.json 46ms (unchanged)
.eslintrc.json 10ms (unchanged)
.github/dependabot.yml 25ms (unchanged)
.github/ISSUE_TEMPLATE/config.yml 3ms (unchanged)
.github/nitpicks.yml 4ms (unchanged)
.github/workflows/ci.yml 9ms (unchanged)
.github/workflows/release_please.yml 4ms (unchanged)
.github/workflows/review_request.yml 4ms (unchanged)
.github/workflows/stale.yml 7ms (unchanged)
.github/workflows/update_deps.yml 5ms (unchanged)
.prettierrc 7ms (unchanged)
.release-please-manifest.json 1ms (unchanged)
package.json 4ms (unchanged)
README.md 42ms (unchanged)
release-please-config.json 2ms (unchanged)
scripts/configureKeyDump.js 18ms (unchanged)
scripts/generateDevicesIndex.js 13ms (unchanged)
scripts/modernExtendRefactor.ts 123ms (unchanged)
scripts/tuyaWhiteLabelGen.js 18ms (unchanged)
src/converters/fromZigbee.ts 799ms (unchanged)
src/converters/toZigbee.ts 585ms (unchanged)
src/devices/acova.ts 59ms (unchanged)
src/devices/acuity_brands_lighting.ts 48ms (unchanged)
src/devices/adeo.ts 26ms (unchanged)
src/devices/adurosmart.ts 9ms (unchanged)
src/devices/aeotec.ts 6ms (unchanged)
src/devices/airam.ts 5ms (unchanged)
src/devices/airzone_aidoo.ts 4ms (unchanged)
src/devices/ajax_online.ts 5ms (unchanged)
src/devices/akuvox.ts 5ms (unchanged)
src/devices/alchemy.ts 3ms (unchanged)
src/devices/aldi.ts 3ms (unchanged)
src/devices/alecto.ts 5ms (unchanged)
src/devices/anchor.ts 3ms (unchanged)
src/devices/atlantic.ts 10ms (unchanged)
src/devices/atsmart.ts 2ms (unchanged)
src/devices/aubess.ts 3ms (unchanged)
src/devices/aurora_lighting.ts 36ms (unchanged)
src/devices/automaton.ts 5ms (unchanged)
src/devices/avatto.ts 4ms (unchanged)
src/devices/awox.ts 13ms (unchanged)
src/devices/axis.ts 4ms (unchanged)
src/devices/bankamp.ts 2ms (unchanged)
src/devices/bega.ts 4ms (unchanged)
src/devices/belkin.ts 2ms (unchanged)
src/devices/bitron.ts 22ms (unchanged)
src/devices/bituo_technik.ts 6ms (unchanged)
src/devices/blaupunkt.ts 4ms (unchanged)
src/devices/blitzwolf.ts 3ms (unchanged)
src/devices/bosch.ts 159ms (unchanged)
src/devices/bouffalo_lab.ts 2ms (unchanged)
src/devices/brimate.ts 2ms (unchanged)
src/devices/bseed.ts 3ms (unchanged)
src/devices/bticino.ts 8ms (unchanged)
src/devices/busch_jaeger.ts 9ms (unchanged)
src/devices/byun.ts 2ms (unchanged)
src/devices/calex.ts 3ms (unchanged)
src/devices/candeo.ts 3ms (unchanged)
src/devices/casaia.ts 4ms (unchanged)
src/devices/cel.ts 2ms (unchanged)
src/devices/centralite.ts 29ms (unchanged)
src/devices/cleode.ts 2ms (unchanged)
src/devices/cleverio.ts 4ms (unchanged)
src/devices/climax.ts 13ms (unchanged)
src/devices/commercial_electric.ts 2ms (unchanged)
src/devices/cree.ts 2ms (unchanged)
src/devices/ctm.ts 114ms (unchanged)
src/devices/current_products_corp.ts 3ms (unchanged)
src/devices/custom_devices_diy.ts 87ms (unchanged)
src/devices/cy_lighting.ts 2ms (unchanged)
src/devices/danalock.ts 4ms (unchanged)
src/devices/danfoss.ts 43ms (unchanged)
src/devices/databyte.ts 6ms (unchanged)
src/devices/datek.ts 27ms (unchanged)
src/devices/dawon_dns.ts 20ms (unchanged)
src/devices/develco.ts 83ms (unchanged)
src/devices/digi.ts 2ms (unchanged)
src/devices/diyruz.ts 27ms (unchanged)
src/devices/dlink.ts 4ms (unchanged)
src/devices/dnake.ts 2ms (unchanged)
src/devices/dresden_elektronik.ts 4ms (unchanged)
src/devices/easyaccess.ts 4ms (unchanged)
src/devices/easyiot.ts 8ms (unchanged)
src/devices/eatonhalo_led.ts 4ms (unchanged)
src/devices/echostar.ts 3ms (unchanged)
src/devices/ecodim.ts 11ms (unchanged)
src/devices/ecolink.ts 3ms (unchanged)
src/devices/ecosmart.ts 9ms (unchanged)
src/devices/ecozy.ts 4ms (unchanged)
src/devices/edp.ts 4ms (unchanged)
src/devices/efekta.ts 41ms (unchanged)
src/devices/eglo.ts 6ms (unchanged)
src/devices/elko.ts 11ms (unchanged)
src/devices/enbrighten.ts 5ms (unchanged)
src/devices/enocean.ts 6ms (unchanged)
src/devices/envilar.ts 5ms (unchanged)
src/devices/essentialb.ts 5ms (unchanged)
src/devices/eucontrols.ts 2ms (unchanged)
src/devices/eurotronic.ts 8ms (unchanged)
src/devices/evanell.ts 6ms (unchanged)
src/devices/evn.ts 3ms (unchanged)
src/devices/evology.ts 3ms (unchanged)
src/devices/evvr.ts 2ms (unchanged)
src/devices/ewelink.ts 9ms (unchanged)
src/devices/ezex.ts 4ms (unchanged)
src/devices/fantem.ts 9ms (unchanged)
src/devices/feibit.ts 28ms (unchanged)
src/devices/fireangel.ts 3ms (unchanged)
src/devices/frankever.ts 3ms (unchanged)
src/devices/frient.ts 3ms (unchanged)
src/devices/futurehome.ts 7ms (unchanged)
src/devices/ge.ts 6ms (unchanged)
src/devices/gewiss.ts 3ms (unchanged)
src/devices/gidealed.ts 2ms (unchanged)
src/devices/giderwel.ts 2ms (unchanged)
src/devices/giex.ts 7ms (unchanged)
src/devices/girier.ts 3ms (unchanged)
src/devices/gledopto.ts 67ms (unchanged)
src/devices/gmmts.ts 110ms (unchanged)
src/devices/gmy.ts 2ms (unchanged)
src/devices/gs.ts 6ms (unchanged)
src/devices/halemeier.ts 7ms (unchanged)
src/devices/hampton_bay.ts 3ms (unchanged)
src/devices/heatit.ts 2ms (unchanged)
src/devices/heiman.ts 72ms (unchanged)
src/devices/heimgard_technologies.ts 14ms (unchanged)
src/devices/hej.ts 4ms (unchanged)
src/devices/hfh.ts 1ms (unchanged)
src/devices/hilux.ts 4ms (unchanged)
src/devices/hive.ts 44ms (unchanged)
src/devices/hommyn.ts 3ms (unchanged)
src/devices/honyar.ts 10ms (unchanged)
src/devices/hornbach.ts 5ms (unchanged)
src/devices/hzc_electric.ts 3ms (unchanged)
src/devices/hzc.ts 4ms (unchanged)
src/devices/icasa.ts 6ms (unchanged)
src/devices/idinio.ts 2ms (unchanged)
src/devices/ihorn.ts 5ms (unchanged)
src/devices/ikea.ts 43ms (unchanged)
src/devices/ilightsin.ts 2ms (unchanged)
src/devices/iluminize.ts 14ms (unchanged)
src/devices/ilux.ts 3ms (unchanged)
src/devices/imhotepcreation.ts 15ms (unchanged)
src/devices/immax.ts 17ms (unchanged)
src/devices/imou.ts 3ms (unchanged)
src/devices/index.ts 18ms (unchanged)
src/devices/innr.ts 45ms (unchanged)
src/devices/inovelli.ts 136ms (unchanged)
src/devices/insta.ts 17ms (unchanged)
src/devices/iolloi.ts 5ms (unchanged)
src/devices/iotperfect.ts 2ms (unchanged)
src/devices/iris.ts 20ms (unchanged)
src/devices/istar.ts 3ms (unchanged)
src/devices/ITCommander.ts 4ms (unchanged)
src/devices/jasco.ts 2ms (unchanged)
src/devices/javis.ts 4ms (unchanged)
src/devices/jethome.ts 6ms (unchanged)
src/devices/jiawen.ts 3ms (unchanged)
src/devices/jumitech.ts 2ms (unchanged)
src/devices/jxuan.ts 3ms (unchanged)
src/devices/kami.ts 2ms (unchanged)
src/devices/keen_home.ts 7ms (unchanged)
src/devices/klikaanklikuit.ts 2ms (unchanged)
src/devices/kmpcil.ts 12ms (unchanged)
src/devices/konke.ts 10ms (unchanged)
src/devices/ksentry.ts 3ms (unchanged)
src/devices/kurvia.ts 2ms (unchanged)
src/devices/kwikset.ts 9ms (unchanged)
src/devices/lanesto.ts 2ms (unchanged)
src/devices/lds.ts 2ms (unchanged)
src/devices/led_trading.ts 6ms (unchanged)
src/devices/ledvance.ts 15ms (unchanged)
src/devices/leedarson.ts 8ms (unchanged)
src/devices/legrand.ts 47ms (unchanged)
src/devices/lellki.ts 11ms (unchanged)
src/devices/letsled.ts 1ms (unchanged)
src/devices/letv.ts 2ms (unchanged)
src/devices/leviton.ts 8ms (unchanged)
src/devices/lg.ts 2ms (unchanged)
src/devices/lidl.ts 70ms (unchanged)
src/devices/lifecontrol.ts 9ms (unchanged)
src/devices/lightsolutions.ts 2ms (unchanged)
src/devices/linkind.ts 13ms (unchanged)
src/devices/linptech.ts 9ms (unchanged)
src/devices/livingwise.ts 3ms (unchanged)
src/devices/livolo.ts 29ms (unchanged)
src/devices/lixee.ts 125ms (unchanged)
src/devices/lonsonho.ts 32ms (unchanged)
src/devices/ls.ts 5ms (unchanged)
src/devices/lubeez.ts 3ms (unchanged)
src/devices/lumi.ts 277ms (unchanged)
src/devices/lupus.ts 8ms (unchanged)
src/devices/lutron.ts 8ms (unchanged)
src/devices/lux.ts 4ms (unchanged)
src/devices/lytko.ts 56ms (unchanged)
src/devices/m_elec.ts 3ms (unchanged)
src/devices/makegood.ts 4ms (unchanged)
src/devices/matcall_bv.ts 2ms (unchanged)
src/devices/meazon.ts 5ms (unchanged)
src/devices/mercator.ts 16ms (unchanged)
src/devices/miboxer.ts 4ms (unchanged)
src/devices/micromatic.ts 1ms (unchanged)
src/devices/modular.ts 2ms (unchanged)
src/devices/moes.ts 50ms (unchanged)
src/devices/muller_licht.ts 12ms (unchanged)
src/devices/multiterm.ts 7ms (unchanged)
src/devices/namron.ts 69ms (unchanged)
src/devices/nanoleaf.ts 2ms (unchanged)
src/devices/neo.ts 31ms (unchanged)
src/devices/net2grid.ts 3ms (unchanged)
src/devices/netvox.ts 2ms (unchanged)
src/devices/nexelec.ts 2ms (unchanged)
src/devices/niko.ts 33ms (unchanged)
src/devices/ninja_blocks.ts 3ms (unchanged)
src/devices/niviss.ts 2ms (unchanged)
src/devices/nodon.ts 13ms (unchanged)
src/devices/nordtronic.ts 3ms (unchanged)
src/devices/nous.ts 8ms (unchanged)
src/devices/novo.ts 3ms (unchanged)
src/devices/nue_3a.ts 17ms (unchanged)
src/devices/nyce.ts 4ms (unchanged)
src/devices/onenuo.ts 5ms (unchanged)
src/devices/onesti.ts 11ms (unchanged)
src/devices/openlumi.ts 2ms (unchanged)
src/devices/orvibo.ts 21ms (unchanged)
src/devices/osram.ts 28ms (unchanged)
src/devices/oujiabao.ts 2ms (unchanged)
src/devices/owon.ts 36ms (unchanged)
src/devices/ozsmartthings.ts 3ms (unchanged)
src/devices/paul_neuhaus.ts 6ms (unchanged)
src/devices/paulmann.ts 13ms (unchanged)
src/devices/peq.ts 3ms (unchanged)
src/devices/perenio.ts 26ms (unchanged)
src/devices/philips.ts 179ms (unchanged)
src/devices/plaid.ts 3ms (unchanged)
src/devices/plugwise.ts 10ms (unchanged)
src/devices/profalux.ts 8ms (unchanged)
src/devices/prolight.ts 4ms (unchanged)
src/devices/pushok.ts 10ms (unchanged)
src/devices/qa.ts 19ms (unchanged)
src/devices/qmotion.ts 3ms (unchanged)
src/devices/qoto.ts 4ms (unchanged)
src/devices/quotra.ts 2ms (unchanged)
src/devices/rademacher.ts 3ms (unchanged)
src/devices/raex.ts 2ms (unchanged)
src/devices/rgb_genie.ts 9ms (unchanged)
src/devices/robb.ts 19ms (unchanged)
src/devices/roome.ts 2ms (unchanged)
src/devices/rtx.ts 6ms (unchanged)
src/devices/salus_controls.ts 8ms (unchanged)
src/devices/samotech.ts 4ms (unchanged)
src/devices/saswell.ts 5ms (unchanged)
src/devices/sber.ts 4ms (unchanged)
src/devices/scanproducts.ts 2ms (unchanged)
src/devices/schlage.ts 2ms (unchanged)
src/devices/schneider_electric.ts 145ms (unchanged)
src/devices/schwaiger.ts 12ms (unchanged)
src/devices/seastar_intelligence.ts 5ms (unchanged)
src/devices/securifi.ts 4ms (unchanged)
src/devices/sengled.ts 15ms (unchanged)
src/devices/sercomm.ts 10ms (unchanged)
src/devices/shade_control.ts 3ms (unchanged)
src/devices/shenzhen_homa.ts 6ms (unchanged)
src/devices/shinasystem.ts 72ms (unchanged)
src/devices/siglis.ts 26ms (unchanged)
src/devices/sikom.ts 4ms (unchanged)
src/devices/sinope.ts 128ms (unchanged)
src/devices/siterwell.ts 6ms (unchanged)
src/devices/skydance.ts 7ms (unchanged)
src/devices/slv.ts 3ms (unchanged)
src/devices/smart_home_pty.ts 2ms (unchanged)
src/devices/smart9.ts 3ms (unchanged)
src/devices/smartenit.ts 4ms (unchanged)
src/devices/smartthings.ts 32ms (unchanged)
src/devices/smartwings.ts 3ms (unchanged)
src/devices/smlight.ts 2ms (unchanged)
src/devices/sohan_electric.ts 2ms (unchanged)
src/devices/solaredge.ts 2ms (unchanged)
src/devices/soma.ts 2ms (unchanged)
src/devices/somfy.ts 4ms (unchanged)
src/devices/somgoms.ts 3ms (unchanged)
src/devices/sonoff.ts 101ms (unchanged)
src/devices/sowilo.ts 11ms (unchanged)
src/devices/spotmau.ts 3ms (unchanged)
src/devices/stelpro.ts 17ms (unchanged)
src/devices/sunricher.ts 42ms (unchanged)
src/devices/swann.ts 2ms (unchanged)
src/devices/sylvania.ts 9ms (unchanged)
src/devices/tapestry.ts 2ms (unchanged)
src/devices/tci.ts 2ms (unchanged)
src/devices/technicolor.ts 5ms (unchanged)
src/devices/terncy.ts 6ms (unchanged)
src/devices/the_light_group.ts 5ms (unchanged)
src/devices/third_reality.ts 19ms (unchanged)
src/devices/titan_products.ts 3ms (unchanged)
src/devices/tlwglobal.ts 2ms (unchanged)
src/devices/tplink.ts 4ms (unchanged)
src/devices/trust.ts 8ms (unchanged)
src/devices/tubeszb.ts 3ms (unchanged)
src/devices/tuya.ts 860ms (unchanged)
src/devices/ubisys.ts 92ms (unchanged)
src/devices/uhome.ts 3ms (unchanged)
src/devices/universal_electronics_inc.ts 10ms (unchanged)
src/devices/vbled.ts 3ms (unchanged)
src/devices/vesternet.ts 20ms (unchanged)
src/devices/viessmann.ts 7ms (unchanged)
src/devices/villeroy_boch.ts 2ms (unchanged)
src/devices/vimar.ts 7ms (unchanged)
src/devices/visonic.ts 8ms (unchanged)
src/devices/vrey.ts 3ms (unchanged)
src/devices/wally.ts 3ms (unchanged)
src/devices/waxman.ts 4ms (unchanged)
src/devices/weiser.ts 6ms (unchanged)
src/devices/weten.ts 7ms (unchanged)
src/devices/wirenboard.ts 41ms (unchanged)
src/devices/wisdom.ts 1ms (unchanged)
src/devices/woolley.ts 4ms (unchanged)
src/devices/woox.ts 8ms (unchanged)
src/devices/wyze.ts 4ms (unchanged)
src/devices/xinghuoyuan.ts 2ms (unchanged)
src/devices/xyzroe.ts 43ms (unchanged)
src/devices/yale.ts 44ms (unchanged)
src/devices/ynoa.ts 6ms (unchanged)
src/devices/yookee.ts 3ms (unchanged)
src/devices/ysrsai.ts 3ms (unchanged)
src/devices/zemismart.ts 32ms (unchanged)
src/devices/zen.ts 5ms (unchanged)
src/devices/zigbeetlc.ts 11ms (unchanged)
src/devices/zipato.ts 2ms (unchanged)
src/index.ts 43ms (unchanged)
src/lib/color.ts 72ms (unchanged)
src/lib/configureKey.ts 37ms (unchanged)
src/lib/constants.ts 13ms (unchanged)
src/lib/develco.ts 15ms (unchanged)
src/lib/ewelink.ts 3ms (unchanged)
src/lib/exposes.ts 137ms (unchanged)
src/lib/generateDefinition.ts 40ms (unchanged)
src/lib/ikea.ts 88ms (unchanged)
src/lib/kelvinToXy.ts 199ms (unchanged)
src/lib/ledvance.ts 6ms (unchanged)
src/lib/legacy.ts 646ms (unchanged)
src/lib/legrand.ts 16ms (unchanged)
src/lib/light.ts 9ms (unchanged)
src/lib/logger.ts 3ms (unchanged)
src/lib/lumi.ts 464ms (unchanged)
src/lib/modernExtend.ts 197ms (unchanged)
src/lib/ota/common.ts 88ms (unchanged)
src/lib/ota/gmmts.ts 5ms (unchanged)
src/lib/ota/index.ts 2ms (unchanged)
src/lib/ota/inovelli.ts 6ms (unchanged)
src/lib/ota/jethome.ts 6ms (unchanged)
src/lib/ota/ledvance.ts 5ms (unchanged)
src/lib/ota/lixee.ts 5ms (unchanged)
src/lib/ota/OTA_URLs.md 20ms (unchanged)
src/lib/ota/salus.ts 7ms (unchanged)
src/lib/ota/securifi.ts 4ms (unchanged)
src/lib/ota/tradfri.ts 6ms (unchanged)
src/lib/ota/ubisys.ts 6ms (unchanged)
src/lib/ota/zigbeeOTA.ts 13ms (unchanged)
src/lib/philips.ts 48ms (unchanged)
src/lib/reporting.ts 24ms (unchanged)
src/lib/store.ts 4ms (unchanged)
src/lib/tuya.ts 193ms (unchanged)
src/lib/types.ts 25ms (unchanged)
src/lib/ubisys.ts 23ms (unchanged)
src/lib/utils.ts 73ms (unchanged)
src/lib/zosung.ts 19ms (unchanged)
test/color.test.js 44ms (unchanged)
test/colortemp_range_missing_allowed.ts 4ms (unchanged)
test/fromZigbee.test.js 10ms (unchanged)
test/generateDefinition.test.ts 27ms (unchanged)
test/index.test.js 53ms (unchanged)
test/lumi.test.js 24ms (unchanged)
test/modernExtend.test.ts 60ms (unchanged)
test/ota.test.ts 22ms (unchanged)
test/otaCommon.test.ts 8ms (unchanged)
test/philips.test.js 9ms (unchanged)
test/sonoff.test.ts 16ms (unchanged)
test/utils.test.js 11ms (unchanged)
test/utils.ts 22ms (unchanged)
tsconfig.json 3ms (unchanged)

File: package-lock.json [hunk 1]
```diff
@@ -1766,7 +1766,6 @@
             "resolved": "https://registry.npmjs.org/@types/node/-/node-22.5.4.tgz",
             "integrity": "sha512-FDuKUJQm/ju9fT/SeX/6+gBzoPzlVCzfzmGkwKvRHQVxi4BntVbyIwf6a4Xn62mrvndLiml6z/UBXIdEVjQLXg==",
             "dev": true,
-            "license": "MIT",
```
Comment (Koenkk) [inline package-lock.json:1769]: Could you revert this change?

File: src/devices/tuya.ts [hunk 2]
```diff
@@ -11591,6 +11591,36 @@ const definitions: DefinitionWithExtend[] = [
             ],
         },
     },
+    {
+        fingerprint: tuya.fingerprint('TS0601', ['_TZE200_abatw3kj']),
```
Comment (Koenkk) [inline src/devices/tuya.ts:11595]: I think this can be merged with `TS0601_din_4`

**Comment by vitalybibikov** [comment]
Actually, don't know how to fix this "linter" without committing 381 file that it changes
